bundler/spec/commands/newgem_spec.rb

```diff
@@ -158,6 +159,26 @@ def bundle_exec_rubocop
     end
   end
 
+  shared_examples_for "--changelog flag" do
+    before do
+      bundle! "gem #{gem_name} --changelog"
+    end
+    it "generates a gem skeleton with a CHANGELOG", :readline do
+      gem_skeleton_assertions
+      expect(bundled_app("#{gem_name}/CHANGELOG.md")).to exist
+    end
+  end
+
+  shared_examples_for "--no-changelog flag" do
+    before do
+      bundle! "gem #{gem_name} --no-changelog"
+    end
+    it "generates a gem skeleton without a CHANGELOG", :readline do
+      gem_skeleton_assertions
+      expect(bundled_app("#{gem_name}/CHANGELOG.md")).to_not exist
+    end
+  end
```

Comment:
@jlgoedert You're defining shared examples, but not actually using them! You need to `it_behaves_like "--changelog flag"` and `it_behaves_like "--no-changelog flag"`somewhere. Have a look at how it's done for `--rubocop` for example.

Reply:
@deivid-rodriguez Nice catch! Actually testing this revealed a couple small issues that I corrected. As always, thanks for the feedback.